_source/logzio_collections/_log-sources/salesforce-commerce-cloud.md

```diff
@@ -0,0 +1,140 @@
+| SFCC_CLIENT_ID        | Client id related to the account that you need to send logs from. [Learn more](https://documentation.b2c.commercecloud.salesforce.com/DOC3/index.jsp?topic=%2Fcom.demandware.dochelp%2Fcontent%2Fb2c_commerce%2Ftopics%2Faccount_manager%2Fb2c_account_manager_add_api_client_id.html)                                                           |      Required |
+| SFCC_LOG_SOURCE    | Flag to represent which log types you would like to collect. `operational` for operational logs, `security` for security logs, `all` for both of them.                                                       |      Required |
+| SFCC_CLIENT_SECRET    | Client secret related to the account that you need to send logs from. [Learn more](https://documentation.b2c.commercecloud.salesforce.com/DOC3/index.jsp?topic=%2Fcom.demandware.dochelp%2Fcontent%2Fb2c_commerce%2Ftopics%2Faccount_manager%2Fb2c_account_manager_add_api_client_id.html)                                                       |      Required |
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours, and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
+
+If you still don't see your logs, see [log shipping troubleshooting]({{site.baseurl}}/user-guide/log-shipping/log-shipping-troubleshooting.html).
+
+</div>
+
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="config">
+
+The default configuration uses a Docker container with environment variables defined by a configuration file.
+
+<div class="tasklist">
+
+
+##### Create a local directory for this integration
+
+You will need a dedicated directory to use it as mounted directory for the Docker container of the Logz.io Salesforce Commerce Cloud.
+
+```shell
+mkdir sfcc-logs-fetcher
+cd sfcc-logs-fetcher
+```
+
+##### Create a configuration file
+
+In the directory created in the previous step, create a file `variables.env` as follows:
+
+```env
+# Environment file for Logz.io Logs for SalesForce Commerce Cloud
+LOGZIO_SHIPPING_TOKEN=<<LOG-SHIPPING-TOKEN>>
+SFCC_HOSTNAME=<<your_sfcc_host>>
+SFCC_CLIENT_ID=<<your_sfcc_client_id>>
+SFCC_CLIENT_SECRET=<<your_sfcc_client_secret>>
+CCTAIL_ARGS=sfcc
+LOGZIO_LISTENER_URL=https://listener.logz.io:8071
+```
+
+Replace the variable values as per the table below:
+
+| Parameter | Description | Required/Default |
+|---|---|---|
+| LOGZIO_SHIPPING_TOKEN | `<<LOG-SHIPPING-TOKEN>>` Your Logz.io account token. {% include log-shipping/log-shipping-token.html %}  | Required  |
+| LOGZIO_LISTENER_URL | {% include log-shipping/listener-var.md %} | Required |
+| SFCC_HOSTNAME         | Hostname from what host need to send logs (ex. `dev01-mysandbox.demandware.net`)                                                                                                                                                                                                                                                                |      Required |
```

Comment:
Also here need to re-name some parameter from SFCC_HOSTNAME to SFCC_SERVER_NAME

Reply:
Do we need to change the description as well? Instead of "hostname from what host"?